src/mochaCustomStackTraceReporter.ts

```diff
@@ -9,12 +9,19 @@ class MochaCustomStackTraceReporter extends reporters.Base
         {
             error.stack = MochaCustomStackTraceReporter.stackTrace(error);
         });
-    }
+        
+        runner.on("start", () => {
+            process.on("unhandledRejection", (reason, promise) => {
+                console.log("ERROR: Unhandled Promise Rejection at: %s", reason.stack || reason);
```

Comment:
unhandled promise rejection at...

Reply:
the execution failed duto ti unhandled promise rejection, details: %s

Reply:
updated